samples/excel/85-preview-apis/chart-data-source.yaml

```diff
@@ -0,0 +1,129 @@
+order: 6
+id: excel-chart-data-source
+name: Chart series data source
+description: This sample shows how to get information about the data source of a chart series.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#log-chart-series-source").click(() => tryCatch(logChartSeriesSource));
+
+        async function logChartSeriesSource() {
+          await Excel.run(async (context) => {
+            // This method retrieves the data source string and type values from
+            // the chart on the Sample worksheet.
+            const sheet = context.workbook.worksheets.getItem("Sample");
+
+            // Get the series collection from the first chart on the worksheet, and
+            // then get the first series in that collection.
+            const seriesCollection = sheet.charts.getItemAt(0).series;
+            const series = seriesCollection.getItemAt(0);
+
+            // Get the series data source string and type values. 
+            const dataSourceString = series.getDimensionDataSourceString("Values");
+            const dataSourceType = series.getDimensionDataSourceType("Values");
+            await context.sync();
+            
+            // Log the data source information to the console.
+            console.log("Qtr1 data source string: " + dataSourceString.value);
+            console.log("Qtr1 data source type: " + dataSourceType.value);
```

Comment:
Since you have the chart series, consider replacing Qtr1 with `series.name` (be sure to add a load statement before the `context.sync`).